scripts/_common.sh

```diff
@@ -25,6 +25,70 @@ function set_permissions {
 # EXPERIMENTAL HELPERS
 #=================================================
 
+#=================================================
+# EXPERIMENTAL HELPERS
+#=================================================
+
+# Create a dedicated fail2ban config (jail and filter conf files)
+#
+# usage: ynh_add_fail2ban_config log_file filter [max_retry [ports]]
+# | arg: log_file - Log file to be checked by fail2ban
+# | arg: failregex - Failregex to be looked for by fail2ban
+# | arg: max_retry - Maximum number of retries allowed before banning IP address - default: 3
+# | arg: ports - Ports blocked for a banned IP address - default: http,https
+ynh_add_fail2ban_config_temp () {
```

Comment:
the CI says : 
```
ynh_remove_fail2ban_config is now an official helper since version '3.5.0'
```
You may remove this part

Reply:
I didn't know that we had an official helper for that…
I'll change it, but I will probably need to try it as I don't know how this new one works…